docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+
+A basis is stable and allows you to re-run unmodified queries indefinitely. This is useful for debugging, auditing, and exposing application data for processing in downstream systems (generating reports, analytics etc.)
+
+If you do not specify a basis then the latest available basis is used by default.
+
+Within XTDB's SQL syntax, specifying the basis can be accomplished by prefixing a query with some [special syntax](/reference/main/sql/queries) which uses the underlying notion of "DEFAULT SYSTEM_TIME" (discussed in the next section):
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q={`
+SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2020-01-01'
+SELECT * FROM PEOPLE`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2020-01-01'
+SELECT * FROM PEOPLE;
+ _id | info | likes | name
+-----+------+-------+------
+(0 rows)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+Here we see that with the basis now set to 2020 no results are returned.
+
+Now try setting the basis to a point in time shortly after our initial transaction:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q={`
+SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2024-01-01' -- try adding 3 days
+SELECT * FROM PEOPLE`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+  <Aside type="caution">The timestamp shown here is for illustration only, you should refer to the output of your recent transactions query for a suitable timestamp</Aside>
+```sql
+user=> SETTING DEFAULT SYSTEM_TIME TO AS OF TIMESTAMP '2024-07-15T12:37:00'
+SELECT * FROM PEOPLE;
+ _id | info | likes | name
+-----+------+-------+------
+   6 |      |       | fred
+(1 row)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+Here we see the original version of the `fred` row, and the `bob` row doesn't exist yet.
+
+### System-Time Columns: automatic time-versioning of rows without audit tables
+
+The mechanism underpinning the basis concept is called 'System Time'. This is what makes data XTDB immutable.
+
+Normally a SQL database will irreversibly lose access to prior states of data after transactions containing `UPDATE` or `DELETE` statements are committed. System-time versioning is a standard defined in [SQL:2011](https://en.wikipedia.org/wiki/SQL:2011), but unlike other implementations, XTDB has it baked into the core of the database engine such that all data is versioned by default and all modification operations are non-destructive.
+
+This minimises the circumstances where developers need to reach for backups or ETL integrations with other warehousing systems in order to recover data. It also helps avoid complicating application schemas with things like "soft delete" columns and audit tables.
```

Comment:
```
XTDB has the [SQL:2011](https://en.wikipedia.org/wiki/SQL:2011) notion of
'System Time' baked into the core of it's engine. This means that all `UPDATE`,
`DELETE` and `INSERT`s are automatically versioned, you'll never lose data
again!

Because this is baked into the database instead of reaching for backups,
"soft delete" columns, audit tables and ETL integrations with other warehousing
systems you can use ordinary(ish) SQL to recover and analyse past data:
```